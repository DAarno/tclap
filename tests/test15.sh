#!/bin/sh
# failure
`dirname $0`/simple-test.sh `basename $0 .sh` test3  --stringTest bbb homer marge bart -- -hv two
