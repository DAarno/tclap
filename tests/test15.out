PARSE ERROR: Argument: (--stringTest)
             Couldn't find delimiter for this argument!

Brief USAGE: 
   ../examples/test3  [-Bh] [--version] [-f=float ...] [-i=int ...]
                      --stringTest=string unTest1 unTest2 [fileName ...]

For complete USAGE and HELP type: 
   ../examples/test3 --help

